docs/DeveloperGuide.md

```diff
@@ -256,7 +256,18 @@ The `Model`, consists of 5 different types of lists.
 <br>
 
 ### 4.5 Storage component
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; <img src="images/StorageUML.png">
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; <img src="images/UpdatedStorageUML.png">
```

Comment:
The colour is slightly different from Alex's. Gradient only applicable for the header. The body should not have gradient. Low priority changes.